migration-engine/qe-setup/src/lib.rs

```diff
@@ -138,13 +138,14 @@ async fn diff_and_apply(schema: &str) {
     .await
     .unwrap();
     let migrations = host.printed.lock().unwrap();
-    let migration = &migrations[0];
+    let migration = migrations[0].clone();
+    drop(migrations);
```

Comment:
Here, is `drop(migrations)` needed to implicitly unlock `host.printed`?

Reply:
Because this is in our test it isn't really needed. But we drop the Guard lock here before the await so that the mutex can be used in another thread.